django/db/models/fields/related.py

```diff
@@ -2114,25 +2152,30 @@ def __init__(self, to, db_constraint=True, swappable=True, **kwargs):
             # Class names must be ASCII in Python 2.x, so we forcibly coerce it
             # here to break early if there's a problem.
             to = str(to)
-        kwargs['verbose_name'] = kwargs.get('verbose_name', None)
-        kwargs['rel'] = ManyToManyRel(
+
+        if symmetrical is None:
+            symmetrical = (to == RECURSIVE_RELATIONSHIP_CONSTANT)
+
+        if through is not None:
+            assert db_table is None, \
```

Comment:
I've put it on a single line in the latest commit but I prefer when the assertion is standalone, then the message is less distracting. What about parentheses?

I removed backslashes from the other asserts and replaced them with parentheses, but these are multilines so it's less debatable.